lightning/src/ln/channelmonitor.rs

```diff
@@ -2268,16 +2272,17 @@ impl<ChanSigner: ChannelKeys> ChannelMonitor<ChanSigner> {
 	}
 
 	/// Check if any transaction broadcasted is paying fund back to some address we can assume to own
-	fn is_mine(&self, tx: &Transaction) -> Option<SpendableOutputDescriptor> {
+	fn is_mine(&mut self, tx: &Transaction, height: u32) {
+		let mut spendable_output = None;
 		for (i, outp) in tx.output.iter().enumerate() { // There is max one spendable output for any channel tx, including ones generated by us
 			if outp.script_pubkey == self.destination_script {
-				return Some(SpendableOutputDescriptor::StaticOutput {
+				spendable_output =  Some(SpendableOutputDescriptor::StaticOutput {
```

Comment:
Likely makes sense to break; after setting.

Reply:
Was in my intention, forget it while introducing reorg-safety, done